src/components/CryptoSelectForm.js

```diff
@@ -1,20 +1,39 @@
-import React, { Component } from 'react';
+/*  I refactored this to be a sub component for the new SelectionForm component.
+		Once again I realized that to keep the code DRY I needed to refactor the selection
+		components to make them more reusable.  This now gets passed into Selection form
+		to render the proper content for the select form elements.
+		---The main reason why there is still this component and the IntlSelectForm is
+		because I couldn't get the crypto.name to render conditionally while being mapped.
+*/
```

Comment:
I refactored this to be a sub component for the new `SelectionForm` component.
Once again I realized that to keep the code DRY I needed to refactor the selection
components to make them more reusable.  This now gets passed into `SelectionForm`
to render the proper content for the select form elements.
---The main reason why there is still this component and the `IntlSelectForm` is
because I couldn't get the crypto.name to render conditionally while being mapped.  To do this I would have had to refactor a few other things that were out of the scope of adding Radium and thus didn't do it.